[2022-04-21 21:34:27,836] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_pers manual__2022-04-21T21:34:10.461544+00:00 [queued]>
[2022-04-21 21:34:27,859] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_pers manual__2022-04-21T21:34:10.461544+00:00 [queued]>
[2022-04-21 21:34:27,859] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 21:34:27,859] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-21 21:34:27,859] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 21:34:27,871] {taskinstance.py:1264} INFO - Executing <Task(Postgres2File): transfer_cso_pers> on 2022-04-21 21:34:10.461544+00:00
[2022-04-21 21:34:27,876] {standard_task_runner.py:52} INFO - Started process 35756 to run task
[2022-04-21 21:34:27,878] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cso_2295_ipo_2_cso.py', 'transfer_cso_pers', 'manual__2022-04-21T21:34:10.461544+00:00', '--job-id', '941', '--raw', '--subdir', 'DAGS_FOLDER/cso_2295/cso_2295_ipo_2_cso.py', '--cfg-path', '/tmp/tmp4o5w3g3t', '--error-file', '/tmp/tmpm7r45sro']
[2022-04-21 21:34:27,879] {standard_task_runner.py:77} INFO - Job 941: Subtask transfer_cso_pers
[2022-04-21 21:34:27,929] {logging_mixin.py:109} INFO - Running <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_pers manual__2022-04-21T21:34:10.461544+00:00 [running]> on host 22e04adb6bfa
[2022-04-21 21:34:27,973] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cso_2295_ipo_2_cso.py
AIRFLOW_CTX_TASK_ID=transfer_cso_pers
AIRFLOW_CTX_EXECUTION_DATE=2022-04-21T21:34:10.461544+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-21T21:34:10.461544+00:00
[2022-04-21 21:34:27,980] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-21 21:34:28,008] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=cso_2295_ipo_2_cso.py, task_id=transfer_cso_pers, execution_date=20220421T213410, start_date=20220421T213427, end_date=20220421T213428
[2022-04-21 21:34:28,055] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-21 21:34:28,146] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-21 21:34:28,150] {dbapi.py:225} INFO - Running statement: UPDATE list_flow_log
                        SET q_source=2,
                        q_target=2, t_end=NOW(), status=2,
                        optional_key='transfer_cso_pers'
                        where id = 84;, parameters: None
[2022-04-21 21:34:28,151] {dbapi.py:233} INFO - Rows affected: 1
[2022-04-21 21:34:28,163] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
